### DISCUSSION TITLE
How to set transparent windows ?

### DISCUSSION THREAD

**Discussion by karcio** [i3/i3#5645] [Q&A] [answered]
Hi, tried many things. 

Started with: 
1. `opacity: 0.5` in .alacritty.yml 
2. Xresources `URxvt.transparent: true URxvt.shading:20`

I guess second option will work with URXVT terminal - but how can I set for all terminal emulators?

**Comment by okraits** [comment, marked as answer]
You have two options:

1. enable transparency for the terminal emulator you use (look at its documentation).
2. Use a compositor (e.g. picom) to get transparency.

**Reply by karcio** [reply]
thank you, picom works fine :)
